scripts/generate_env.sh

```diff
@@ -1,2 +1,4 @@
 #!/bin/bash -e
-poetry run python scripts/generate_env.py
\ No newline at end of file
+# docker compose requires env file to exist even though we don't use it for this command
```

Comment:
Ah, this is the issue I touched on above - looks like we might just want to update your Readme to use this!